src/html_support_files/odoc.css

```diff
@@ -759,19 +783,29 @@ td.def-doc *:first-child {
   line-height: 1.2;
 }
 
+.odoc-search + * + .odoc-toc {
```

Comment:
A comment saying that this is to accommodate the sidebar would be useful.

Reply:
this is done